lib/tasks/model.rake

```diff
@@ -44,21 +44,23 @@ def consolidate(mappings, dryrun)
       puts "There is a problem with the entry with regular expression #{rex}"
       next
     end
+    # rubocop:disable Lint/UselessAssignment
     original_models = Model.count
+    # rubocop:enable Lint/UselessAssignment
```

Comment:
I think this cop is actually correct, this line should be outside the loop

Reply:
Thanks, should be all good now :bowtie: